man/repart.d.xml

```diff
@@ -426,18 +426,40 @@
       <varlistentry>
         <term><varname>CopyFiles=</varname></term>
 
-        <listitem><para>Takes a pair of colon separated absolute file system paths. The first path refers to
-        a source file or directory on the host, the second path refers to a target in the file system of the
-        newly created partition and formatted file system. This setting may be used to copy files or
-        directories from the host into the file system that is created due to the <varname>Format=</varname>
-        option. If <varname>CopyFiles=</varname> is used without <varname>Format=</varname> specified
-        explicitly, <literal>Format=</literal> with a suitable default is implied (currently
-        <literal>vfat</literal> for <literal>ESP</literal> and <literal>XBOOTLDR</literal> partitions, and
-        <literal>ext4</literal> otherwise, but this may change in the future). This option may be used
-        multiple times to copy multiple files or directories from host into the newly formatted file system.
-        The colon and second path may be omitted in which case the source path is also used as the target
-        path (relative to the root of the newly created file system). If the source path refers to a
-        directory it is copied recursively.</para>
+        <listitem><para>Takes a colon-separated triplet in the form
```

Comment:
Why a new option, instead of just copying the attributes if they are present? We have too many options already...

Reply:
This was the original behavior of this patch.  See this discussion https://github.com/systemd/systemd/pull/35401#discussion_r1863141746 about why it changed.

I also think this is the behavior that makes the most sense.....